### PR TITLE
Update Readme.md

### LINKED ISSUE #241: Make a better README.md
Make a better and more explanatory README.md

### PR DESCRIPTION
**Pull request by sagban**
Closes https://github.com/coala/landing-frontend/issues/241

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Review by jschwarzwalder — APPROVED** [review]

**Comment by sagban** [comment]
@jayvdb can you assigned me some work? According to newcomer guide, now I should work on difficulty/low issue.

**Review by Hori75 — APPROVED** [review]

**Review by hizkifw — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
+To run locally, first clone the repository:
+```sh
     $ git clone https://github.com/coala/landing-frontend.git
     $ cd landing-frontend
-    $ python2 -m SimpleHTTPServer 8080
-
+```
+Then to run the server:
+```sh
+    $ python2 -m SimpleHTTPServer 8080 #For python 2.x
```
Comment (jayvdb) [inline README.md]: We dont want to recommend people use `python2`

People should only use python3 now

File: README.md [hunk 2]
```diff
@@ -1,15 +1,21 @@
 # landing-frontend
-coala Landing Page - https://gitlab.com/coala/landing is needed as backend
-
+This is the coala frontend Landing Page which uses - https://gitlab.com/coala/landing as its backend.
```
Comment (jayvdb) [inline README.md]: This repository is the https://coala.io/ website frontend.
It uses https://gitlab.com/coala/landing as its backend.

**Review by jayvdb — CHANGES_REQUESTED** [review]

**Pull request closed without merging**